include/layer.hpp

```diff
@@ -306,17 +303,17 @@ class ConvLayer : public Layer {
 
   void applyBWD(MyReal *state, MyReal *state_bar, int compute_gradient);
 
-  inline MyReal
-  apply_conv(MyReal *state,   // state vector to apply convolution to
-             int output_conv, // output convolution
-             int j,           // row index
-             int k);          // column index
+  inline MyReal apply_conv(
```

Comment:
Why the style change for these inline functions?

Reply:
https://google.github.io/styleguide/cppguide.html#Namespaces

Reply:
Basically that's the style guide I'm following. There is an example in this section and an explicit section on inline functions.